fcrepo-fixity/src/main/java/org/fcrepo/camel/fixity/FcrepoFixityConfig.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.camel.fixity;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.http.HttpComponent;
+import org.fcrepo.camel.common.config.BasePropsConfig;
+import org.fcrepo.camel.common.config.ConditionOnPropertyTrue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * A configuration class for the Fixity service
+ *
+ * @author dbernstein
+ */
+@Configuration
+@Conditional({FcrepoFixityConfig.FixityEnabled.class})
+public class FcrepoFixityConfig extends BasePropsConfig {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FcrepoFixityConfig.class);
+    static final String FIXITY_ENABLED = "fixity.enabled";
+
+    static class FixityEnabled extends ConditionOnPropertyTrue {
+        FixityEnabled() {
+            super(FcrepoFixityConfig.FIXITY_ENABLED, false);
+        }
+    }
+
+    @Value("${fixity.input.stream:broker:queue:fixity}")
+    private String inputStream;
+
+
+    @Value("${fixity.delay:0}")
+    private long fixityDelay;
+
+    @Value("${fixity.failure:file:/tmp/?fileName=fixityErrors.log&fileExist=Append}")
+    private String fixityFailure;
+
+
+    @Value("${fixity.success:mock:fixity.success}")
+    private String fixitySuccess;
+
+    /**
+     * The input stream for the fixity service
```

Comment:
```suggestion
     * The JMS message stream for the fixity service
```
I have to admit, calling this InputStream really throws me off as a java developer :)

Reply:
I agree.  Will change.

Reply:
Since it is used all over I'll change the docs here for now and create a ticket to change all instances of input.stream to something else.